scripts/deploy/deploy-webapp.ps1

```diff
@@ -27,7 +27,15 @@ param(
     [Parameter(Mandatory=$false)]
     [string]
     # Authority for client applications that are not configured as multi-tenant.
-    $Authority="https://login.microsoftonline.com/common"
+    $Authority="https://login.microsoftonline.com/common",
+
+    [string]
+    # Version to display in UI.
+    $Version = "",
+
+    [string]
+    # Additional information given in version info. (Ex: commit SHA)
+    $InformationalVersion = ""
```

Comment:
can we make the name of this new arg more descriptive? (in the other files as well)
```suggestion
    $CommtiSha = ""
```

Reply:
Well, anything arbitrary string can be passed here. In the deployment script, I just chose to pass the commit SHA for now because I thought it made sense.

Reply:
yeah, that's fair. so maybe `VersionInfo`?

Reply:
Deal! :)